style.css

```diff
@@ -20,32 +20,70 @@ body {
   margin: 0;
   display: flex;
   flex-wrap: wrap;
+  /* Making this fullscreen smhow?. */
+  justify-content: left;
+  align-items: top;
+}
 
-  /* This centers our sketch horizontally. */
-  justify-content: center;
+#floating-button-area{
+  position: absolute;
+  left: 40px;
+  top: 60px;
+  width: 100px;
+  height: 200px;
+  display: flex;
+  flex-wrap: nowrap;
+  justify-content: space-between;
+  flex-direction: column;
+}
 
-  /* This centers our sketch vertically. */
-  align-items: center;
+#key-selection-area {
+  position:absolute;
+  left:50px;
+  bottom: 50px;
+  width:1600px;
```

Comment:
this is too wide. we might have to think about how to make it scale according to the viewport width until some minimum width. narrower than that will qualify for some mobile effort which could be done later